maps/torch/datums/uniforms_fleet.dm

```diff
@@ -3,14 +3,14 @@
 	departments = COM
 
 	utility_under = /obj/item/clothing/under/solgov/utility/fleet/command
-	utility_extra = list(
-				/obj/item/clothing/under/solgov/utility/fleet/combat/command,
-				/obj/item/clothing/head/beret/solgov/fleet/command,
-				/obj/item/clothing/head/ushanka/solgov/fleet,
-				/obj/item/clothing/suit/storage/hooded/wintercoat/solgov/fleet,
-				/obj/item/clothing/head/soft/solgov/fleet,
-				/obj/item/clothing/gloves/thick/duty/solgov/cmd
-			)
+	utility_extra = list(/obj/item/clothing/under/solgov/utility/fleet/officer/command,
```

Comment:
This entire section of indentation is incorrect and one too far, not to mention you have stuff on the same line as the opening and closing parenthesis. Please fix.

Reply:
Also, this applies to all modified lists.